src/command_modules/azure-cli-natgateway/azure/cli/command_modules/natgateway/__init__.py

```diff
@@ -0,0 +1,16 @@
+from azure.cli.core import AzCommandsLoader
+from azure.cli.command_modules.mymod._help import helps  # pylint: disable=unused-import
+
+class NatGatewayLoader(AzCommandsLoader):
+
+    def load_command_table(self, args):
+      from azure.cli.core.commands import CliCommandType
+
+      natgateway_custom = CliCommandType(
+        operations_tmpl='azure.cli.command_modules.maps#NatGatewayOperations.{}',
+      )
+
+      with self.command_group('network natgateway', natgateway_custom) as g:
```

Comment:
natgateway should be hyphenated `nat-gateway`.

Reply:
https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md#command-naming-and-behavior